spec/requests/planning_application_list_spec.rb

```diff
@@ -40,8 +40,8 @@
     end
 
     it "returns a list of serialized planning application" do
-      planning_application_1 = create(:planning_application, :determined)
-      planning_application_2 = create(:planning_application, :determined)
+      planning_application_1 = create(:planning_application).tap { |pa| pa.update_and_timestamp_status(:determined) }
+      planning_application_2 = create(:planning_application).tap { |pa| pa.update_and_timestamp_status(:determined) }
```

Comment:
How about updating factories to update `determined_at` on the `after_create`?

Reply:
Thanks. Yes I'm not sure why I went this long way round. Updated.